.github/actions/pull_images/action.yml

```diff
@@ -9,22 +9,6 @@ inputs:
 runs:
   using: composite
   steps:
-    - name: Free some place for scenarios known to require lot of images
```

Comment:
Did you checked that it was not needed anymore? we add by the past issues because the filesystem was close to be full, with at best huge increase in execution time, and worst, cryptic failures.

Reply:
I didn't see any issues that I can tell from removing this. I wouldn't expect a problem even with several Docker images as the images we're using are not small by any means but they're not that big either. If this is ever a problem again, I'd look into fix it in other ways as this step is extremely slow and should be avoided if at all possible.

Reply:
So, first thing first.

This change is not related to "Run DB startup in parallel". Can you create another PR? it will ease the process for this current PR.

Secondly, this step is not here for nothing, we had a hard time to fix that when it happened, docker errors are cryptic when disk is close to be full. I'm totally supportive to remove it if it's not necessary, but we won't go blindly into that direction. So could you :

1. get the time we save by removing it (maybe one of the three `rm` is taking most of the time?)
2. get the remaining space after all scenario have ran on each weblog before and after the change (well, only the one with the less space is important, but I don't know which weblog take the more space)

Based on that, we'll be able to have an good decision. And if it's a go, we'll keep the code commented, and raise the attention of the R&P team, because if it happen, we must act quickly.